webapp/channels/src/components/integrations/abstract_command.tsx

```diff
@@ -5,29 +5,30 @@ import React, {ChangeEvent} from 'react';
 import {FormattedMessage} from 'react-intl';
 import {Link} from 'react-router-dom';
 
-import {Command} from '@mattermost/types/integrations';
 import BackstageHeader from 'components/backstage/components/backstage_header';
 import Constants from 'utils/constants';
 import * as Utils from 'utils/utils';
 import FormError from 'components/form_error';
 import SpinnerButton from 'components/spinner_button';
 import LocalizedInput from 'components/localized_input/localized_input';
+import ExternalLink from 'components/external_link';
 
 import {t} from 'utils/i18n';
+import {Command} from '@mattermost/types/integrations';
 import {Team} from '@mattermost/types/teams';
 
 const REQUEST_POST = 'P';
 const REQUEST_GET = 'G';
 
 type Props = {
-    team: Team;
-    header: {id: string ; defaultMessage: string};
-    footer: {id: string ; defaultMessage: string};
-    loading: {id: string ; defaultMessage: string};
-    renderExtra?: JSX.Element;
-    serverError: string;
-    initialCommand?: Partial<Command>;
-    action: (command: Command) => Promise<void>;
+    team: Team; // The current team
```

Comment:
@M-ZubairAhmed do we need to keep also multiline commenting for the the tool for documentation or single line comments are also fine?

Reply:
Sure, will make the changes